HandsOn/Group04/mapping/mapping.yml

```diff
@@ -0,0 +1,59 @@
+prefixes:
+  rr: http://www.w3.org/ns/r2rml#
+  ont: http://linkeddata.es/group04/ontology/EstacionesMedidaControlAcustico#
+  dbo: http://dbpedia.org/ontology/
+  res: http://linkeddata.es/group04/resource/
+  rml: http://semweb.mmlab.be/ns/rml#
+  xsd: http://www.w3.org/2001/XMLSchema#
+  
+  mappings:
+  estaciones:
+    sources:
+      - [EstacionesMedidaControlAcustico_mod.csv~csv]
+      s: res:estaciones/$(N)
+       po:
+      - [a, ont:Estacion]
+      - [ont:hasNumber,$(N)]
+      - [ont:hasName,$(Nombre)]
+      - p: ont:isLocated
```

Comment:
here you only can specify two parameters, and as it is the same source in sources, should be the same parameter for str1 and str2, $(COD_VIA)